runtimes/eden/Cargo.toml

```diff
@@ -97,6 +99,7 @@ runtime-benchmarks = [
   "xcm/runtime-benchmarks",
   "xcm-builder/runtime-benchmarks",
   "xcm-executor/runtime-benchmarks",
+  "orml-xtokens/runtime-benchmarks",
```

Comment:
There are no benchmarks for orml-traits ?

Reply:
no benchmarks there